### DISCUSSION TITLE
Error "Cannot access contents of url"

### DISCUSSION THREAD

**Discussion by Briangce** [AutomaApp/automa#941] [General]
Please if anyone can help me I would really appreciate it. This error started with recent automa updates, before everything was working normally. What my workflow does is that it enters a page and what the javascript does is extract 10 links from the page and open them in other tabs, that is repeated 10 times. The error is this "The extensions gallery cannot be scripted.
On the JavaScript code block" this other
"JavaScript code Cannot access contents of url "chrome-extension://infppggnoaenmfagbfknfkancpbljcca/newtab.html#/workflows/7SkA1wsU1QhLcS6j2jHEd?tab=editor". Extension manifest must request permission to access this host."

The javascript code is the next 
var urlCampanhas = [],
    elemHtml = document.querySelectorAll(".table-hover tbody tr td:nth-child(4) a:first-child")

for(let i = 0 ; i < elemHtml.length; i++){
   urlCampanhas.push(elemHtml[i].href) 
  }
  
automaSetVariable('campanhas', urlCampanhas);
console.log(urlCampanhas);
automaNextBlock();

![AAAAAAAAAA](https://user-images.githubusercontent.com/77473583/200837199-74b42d06-ea5a-413d-b4ed-31ba7049505d.png)

**Comment by Kholid060** [comment]
> The error is this "The extensions gallery cannot be scripted.

This error occurs if you're trying to do Automation inside the Chrome Web Store website because an extension can't access that page. 

Instead of using JS Block to get all the URLs, you can replace it with the Attribute Value block with the "Multiple" option enabled.

![image](https://user-images.githubusercontent.com/22908993/200845228-b8d3041b-45ee-4297-bf50-81fb0a2a07bb.png)

And it would be better if you also send the JSON file of the workflow so I can replicate the error.

**Reply by Briangce** [reply]
I'l try your recommendation. Ty for answer
